compute_tools/src/spec.rs

```diff
@@ -152,8 +152,12 @@ pub fn handle_roles(spec: &ComputeSpec, client: &mut Client) -> Result<()> {
             {
                 RoleAction::Update
             } else if let Some(pg_pwd) = &r.encrypted_password {
-                // Check whether password changed or not (trim 'md5:' prefix first)
-                if pg_pwd[3..] != *role.encrypted_password.as_ref().unwrap() {
+                // Check whether password changed or not (trim 'md5' prefix first if any)
+                let pg_pwd = match pg_pwd.strip_prefix("md5") {
```

Comment:
instead of copying the string, I'd recommend slicing it 

```rust
let pg_pwd_new = if pg_pwd.starts_with("md5") {
    &pg_pwd[3..]
} else {
    &pg_pwd
};
```

Reply:
Yeah, looks better. Clippy was complaining on this particular code, so I changed it a bit